Gifski/MainWindowController.swift

```diff
@@ -0,0 +1,175 @@
+import Cocoa
+import ProgressKit
+import DockProgress
+
+extension NSNib.Name {
+	static let mainWindowController = NSNib.Name("MainWindowController")
+}
+
+class MainWindowController: NSWindowController {
+
+	private var progressObserver: NSKeyValueObservation?
+
+	lazy var circularProgress = with(CircularProgressView(frame: CGRect(widthHeight: 160))) {
+		$0.foreground = .appTheme
+		$0.strokeWidth = 2
+		$0.percentLabelLayer.setAutomaticContentsScale()
+		$0.percentLabelLayer.implicitAnimations = false
+		$0.layer?.backgroundColor = .clear
+		$0.isHidden = true
+		$0.centerInWindow(window)
+	}
+
+	lazy var videoDropView = with(VideoDropView()) {
+		$0.frame = window!.contentView!.frame
+		$0.dropText = "Drop a Video to Convert to GIF"
+		$0.onComplete = { url in
+			self.convert(url.first!)
+		}
+	}
+
+	var choosenDimensions: CGSize = .zero
```

Comment:
I think this should be an optional instead of `.zero`. You would never want `.zero` dimensions.

Reply:
The reason I use .zero is that it is the default value in Gifski.Conversion, otherwise I would have to use ?? .zero there.

Do you prefer if I use `choosenDimensions ?? .zero` in Gifski.Conversion initialiser instead, or use `nil` as the default param in Gifski.Conversion and setup GifskiWrapper like we did before: https://github.com/sindresorhus/gifski-app/blob/v1.0.0/Gifski/Gifski.swift#L33

Reply:
> or use nil as the default param in Gifski.Conversion and setup GifskiWrapper like we did before: /Gifski/Gifski.swift@v1.0.0#L33

This. I think the intent is clearer that way.